src/main/java/com/rocketden/main/game_object/NotificationType.java

```diff
@@ -0,0 +1,19 @@
+package com.rocketden.main.game_object;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.rocketden.main.exception.NotificationError;
+import com.rocketden.main.exception.api.ApiException;
+
+public enum NotificationType {
+    SUBMIT_CORRECT, SUBMIT_INCORRECT, TEST_CORRECT, CODE_STREAK, TIME_LEFT;
```

Comment:
I think the first component makes sense -- the fact that it's split up on the backend and frontend is strange, though I tried to separate the dynamically changing content (on the backend) and the static content (on the frontend). The reason I didn't try to programmatically do this was because separating the "is X minutes" from "are X minutes" depending on X would require a decent number of checks, but I'm open to implementing this.

Reply:
Ok, I made a map in the `GameTimer` class that handles the content for the `TIME_LEFT` notifications, since they are the main one with this problem. I'm open to further changes though, so I won't resolve this quite yet -- let me know.

<img width="461" alt="GameTimer Duration-Content Map" src="https://user-images.githubusercontent.com/7987279/105567509-30bfa980-5ce7-11eb-8739-4a6feceb83c2.png">